build.gradle

```diff
@@ -635,12 +635,14 @@ subprojects {
         }
 
         // Add tasks to run various checkers on all the main source sets.
-        // TODO: fix or suppress all not.interned warnings and remove the suppression here.
-        createCheckTypeTask(project.name, 'org.checkerframework.checker.interning.InterningChecker', 'Interning', ['-AsuppressWarnings=not.interned'])
-        createCheckTypeTask(project.name, 'org.checkerframework.checker.nullness.NullnessChecker', 'Nullness')
+        // These pass and are run by nonJunitTests.
+        createCheckTypeTask(project.name, 'org.checkerframework.checker.interning.InterningChecker', 'Interning')
```

Comment:
Should we add `-ArequirePrefixInWarningSuppressions`?

Reply:
I will do that in a separate pull request.